fontir/src/source.rs

```diff
@@ -56,16 +56,20 @@ pub trait Source {
     /// Create a function that could be called to generate [crate::ir::StaticMetadata].
     ///
     /// When run work should update [Context] with new [crate::ir::StaticMetadata].
-    fn create_static_metadata_work(&self, context: &Context) -> Result<Box<dyn Work>, Error>;
+    fn create_static_metadata_work(&self, input: &Input) -> Result<Box<dyn Work + Send>, Error>;
 
     /// Create a function that could be called to generate IR for glyphs.
     ///
     /// Batched because some formats require IO to figure out the work.
+    /// Expected to return a Vec aligned with the glyph_names input. That is,
```

Comment:
> Expected to return a Vec aligned with the glyph_names input...

... but `glyph_names` is typed as a HashSet so I believe its order will be kind of abitrary? If the order matters, should it not also be a Vec (or IndexSet)?

Reply:
IndexSet I think, good catch.

Reply:
https://github.com/googlefonts/fontmake-rs/pull/77